crates/bevy_render/src/entity.rs

```diff
@@ -25,6 +25,11 @@ pub struct MeshBundle {
     pub global_transform: GlobalTransform,
 }
 
+/// Marker component used in [PerspectiveCameraBundle] and
+/// [OrthographicCameraBundle].
+#[derive(Default)]
+pub struct GameplayCamera;
```

Comment:
I don't like the name `GameplayCamera`.

I think `MainCamera` would be better, considering that we have the `MainPass` marker component for entities that are to be rendered in the "main pass", which is where this camera is used.

Reply:
Ok, I don't mind one way or the other, I think your reasoning is good though so I can change it to `MainCamera`.

Will ping @alice-i-cecile  though as the original naming came from #1854 .
Any comments?

Reply:
I'd like to avoid `MainCamera:` it becomes very confusing with multiple viewports when you have multiple cameras, all of which have the `MainCamera` marker component.

The idea to communicate here is "renders objects in the game world, not the UI". `WorldCamera` might also makes sense, and is less game-specific, but given that UI entities are also going to be stored in the `World` it could be confusing as well.